gpii/node_modules/flowManager/src/PSPChannel.js

```diff
@@ -78,6 +81,26 @@ fluid.defaults("gpii.pspChannel", {
     members: {
         outputBlocked: null
     },
+    // GPII-3828: this model relay reports default settings to psp clients when model.settingControls is empty.
+    // This will be replaced with GPII-3318 (https://issues.gpii.net/browse/GPII-3318) implementation.
+    modelRelay: {
+        pspChannel: {
+            // Passing the entire that.model rather than that.model.settingControls into the relay function is to
+            // deal with the case when that.model === undefined. Passing that.model.settingControls in this case
+            // will cause model to be auto-converted to undesired empty object.
+            source: "{that}.model",
```

Comment:
Expressing this as a "relay constraint" is an interesting approach but it makes the implementation too much of a mess, with the strange "unescaping" implementation on new line 356. It will lead to a much more direct workflow if you just adjust the values in "schemas" on original line 179 in sessionToPSP - and in fact you can just do this on component startup to avoid any runtime overhead.